nixos/modules/services/monitoring/graphite.nix

```diff
@@ -489,7 +490,10 @@ in {
 
             touch ${dataDir}/db-created
 
-            chown -R graphite:graphite ${cfg.dataDir}
+            chown graphite:graphite ${cfg.dataDir}
+            chown graphite:graphite ${cfg.dataDir}/db-created
+            chown graphite:graphite ${cfg.dataDir}/whisper
+            chown -R graphite:graphite ${cfg.dataDir}/log
```

Comment:
Does `db-created` really needs to be owned by graphite? I would also move the touch command to the last position in case commands before that fail.